renku/cli/workflow.py

```diff
@@ -1058,3 +1058,51 @@ def visualize(sources, columns, exclude_files, ascii, interactive, no_color, pag
         text_output, navigation_data, result.output.vertical_space, use_color=not no_color
     )
     viewer.run()
+
+
+@workflow.command()
+@click.option(
+    "--mapping",
+    metavar="<file>",
+    type=click.Path(exists=True, dir_okay=False),
+    help="YAML file containing parameter mappings to be used.",
+)
+@click.option(
+    "--dry-run",
+    "dry_run",
+    is_flag=True,
+    default=False,
+    help="Print the generated plans with their parameters instead of executing.",
+    show_default=True,
+)
+@click.option(
+    "provider",
+    "-p",
+    "--provider",
+    default="cwltool",
+    show_default=True,
+    type=click.Choice(Proxy(_available_workflow_providers), case_sensitive=False),
+    help="The workflow engine to use.",
+)
+@click.option("config", "-c", "--config", metavar="<config file>", help="YAML file containing config for the provider.")
+@click.argument("name_or_id", required=True)
+def loop(name_or_id, mapping, dry_run, provider, config):
```

Comment:
Command also needs to be documented at the top of this file (as that's what actually ends up on RTD). Ideally with an asciinema recording (i used `asciinema rec -i 5`, with PS1="\$ " in my .bash_profile, a terminal size of 55x102 (hxw) and asciicast2gif to create gifs, e.g. `asciicast2gif -w 102 -h 55 execute_plan.rec execute_plan.gif`)